### ISSUE TITLE
[BUG] Reformatter doesnt align </script> with its <script>

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#93]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Window
 - Python Version: 3.7
 - djLint Version: 0.6.6

## Issue
The reformatter doesn't update this HTML.
```html
<script>
    window.flag = false;
    document.getElementById('FoobarLinkPlaceHolder').style.display = 'none';
    </script>
<a name="foo-bar-{{ sequence.id }}"></a>
```

Removing the `<a...>` at the end, and it will reformat correctly to be
```html
<script>
    window.flag = false;
    document.getElementById('FoobarLinkPlaceHolder').style.display = 'none';
</script>
```

**Comment by jayvdb** [comment]
Another slightly different one.

```html
<html>
    <head>
        <link href="{% static  'foo/bar.css' %}" rel="stylesheet"/>

        <!--JS-->
        <script src="{% static  'foo/bar.js' %}"></script>
    </head>
</html>
```
becomes
```html
<html>
    <head>
        <link href="{% static  'foo/bar.css' %}" rel="stylesheet"/><!--JS-->
    <script src="{% static  'foo/bar.js' %}"></script>
</head>
</html>
```

I add it here, because again changing the "script" to "div" and suddenly it reformats appropriately.

**Comment by christopherpickering** [comment]
It seems to stem from the new regex I was using to strip whitespace.... I had the re.M flag w/out the re.DOTALL and blindly supposed it was not matching newlines. I've changed it to use [ \t] (space or tab) instead of anything white.
